tests/dp_computations_test.py

```diff
@@ -495,6 +497,66 @@ def test_compute_dp_var(self):
                                delta=0.0002)
         self.assertAlmostEqual(np.mean(var_values), expected_var, delta=0.5)
 
+    @parameterized.parameters(
+        {
+            "eps": 2.0,
+            "max_partitions_contributed": 10,
+            "max_contributions_per_partition": 2
+        }, {
+            "eps": 3.5,
+            "max_partitions_contributed": 100,
+            "max_contributions_per_partition": 10
+        })
+    def test_compute_dp_count_noise_params_laplace(
+            self, eps: float, max_partitions_contributed: int,
+            max_contributions_per_partition: int):
+        params = dp_computations.MeanVarParams(
+            eps=eps,
+            delta=0,
+            max_partitions_contributed=max_partitions_contributed,
+            max_contributions_per_partition=max_contributions_per_partition,
+            noise_kind=pipeline_dp.NoiseKind.LAPLACE,
+            min_value=0,
+            max_value=0)
+        expected_scale = max_partitions_contributed * max_contributions_per_partition / eps * np.sqrt(
+            2)
+
+        noise_kind, scale = dp_computations.compute_dp_count_noise_params(
+            params)
+        self.assertEqual("laplace", noise_kind)
+        self.assertAlmostEqual(scale, expected_scale, delta=1e-10)
+
+    @parameterized.parameters(
+        {
+            "eps": 2.0,
+            "delta": 1e-8,
+            "max_partitions_contributed": 10,
+            "max_contributions_per_partition": 2,
+            "expected_scale": 16.787247422534485
+        }, {
+            "eps": 1,
+            "delta": 1e-5,
+            "max_partitions_contributed": 1,
+            "max_contributions_per_partition": 1,
+            "expected_scale": 3.732421875
+        })
+    def test_compute_dp_count_noise_params_gaussian(
+            self, eps: float, delta: float, max_partitions_contributed: int,
+            max_contributions_per_partition: int, expected_scale: float):
+        params = dp_computations.MeanVarParams(
+            eps=eps,
+            delta=delta,
+            min_value=0,
+            max_value=0,
+            max_partitions_contributed=max_partitions_contributed,
+            max_contributions_per_partition=max_contributions_per_partition,
+            noise_kind=pipeline_dp.NoiseKind.GAUSSIAN)
+
+        noise_kind, scale = dp_computations.compute_dp_count_noise_params(
+            params)
+        self.assertEqual("gaussian", noise_kind)
+        self.assertAlmostEqual(scale, expected_scale)
+
 
 if __name__ == '__main__':
-    unittest.main()
+    absltest.main()
```

Comment:
Is this intentional?

Reply:
yes, parametrized tests are in absltest, and not available in unittest, so in this file it's replaced using unittest to absltest